en/micro-integrator/docs/setup/observability/setting-up-classic-observability-deployment.md

```diff
@@ -104,7 +117,65 @@ Follow the steps below to enable statistics for the **endpoint** artifacts:
 1.  Select the required endpoint artifacts from the project explorer. 
 2.  Select **Statistics Enabled** and (if required) **Trace Enabled** as shown below.
      ![endpoint properties](../../assets/img/ei-analytics/endpoint-properties.png)
+     
+### Enabling load balancing Data Agent for Publishing to Analytics
```

Comment:
### Load balancing among multiple Analytics servers

You can send events to multiple Analytics servers either by sending the same event to many Analytics servers or by load balancing events among a set of servers. This handles the failover problem. When events are load balanced within a set of servers and if one receiver cannot be reached, events are automatically sent to the other available and active Analytics servers. The following scenarios are covered in this section.